src/share/classes/com/sun/javatest/regtest/agent/JUnitRunner.java

```diff
@@ -69,6 +79,14 @@ public static void main(ClassLoader loader, String... args) throws Exception {
             cl = JUnitRunner.class.getClassLoader();
         }
         Class<?> mainClass = Class.forName(className, false, cl);
+        if (JUNIT_RUN_WITH_JUNIT_4) {
+            runWithJUnit4(mainClass);
+        } else {
+            runWithJUnitPlatform(mainClass);
+        }
+    }
+
```

Comment:
It's good to have the fallback for a while, but in time (7.1, 7.2) we should remove it.